app/views/admin/overview/index.html.erb

```diff
@@ -0,0 +1,63 @@
+<div class="grid s:grid-cols-3 s:grid-rows-4 gap-3">
+  <article style="grid-column: span 2; grid-row: span 4;">
+    <section class="crayons-card mb-3">
+      <div class="crayons-card__body">
+        <h3 class="crayons-subtitle-2">Welcome Checklist</h3>
+        <p class="crayons-field__description mb-3">Here are things we recommend you do now that you're on Forem</p>
+        <ol>
+          <li>Setup and configure your Forem under <a href="/admin/config">Config</a></li>
+          <li><a href="/admin/invitations">Invite people</a> to join your Forem</li>
+          <li><a href="/admin/tags">Update your Forem tags</a></li>
+          <li><a href="/admin/pages">Create pages</a> and <a href="/admin/navigation_links">sort them in the navigation sidebar</a></li>
+          <li>Read the <a href="https://forem.gitbook.io/forem-admin-guide/quick-start-guide">Quick Start Guide</a></li>
+          <li><a href="/admin/welcome">Set up a Welcome Thread</a></li>
+          <li>Browse the <a href="https://forem.gitbook.io/forem-admin-guide">Forem Admin Guide</a></li>
+          <li>Join the private <a href="https://creators.forem.com">Creators Forem</a>! (email <a href="mailto:support@forem.com">support@forem.com</a> if you need an invite)</li>
+        </ol>
+        <p class="m-3 align-center"><a href="">Visit Forem Admin Guide</a></p>
+      </div>
+    </section>
+    <h2 class="crayons-subtitle-1 mb-3">Resources</h2>
+    <article class="grid s:grid-cols-2 gap-3">
+      <section class="crayons-card">
+        <div class="crayons-card__header">
+          <h3 class="crayons-subtitle-2">Forem Admin Guide</h3>
+        </div>
+        <div class="crayons-card__body flex flex-wrap" style="min-height: 150px;">
+          <p>This guide gives you the best practices on how you can manage and grow your community on Forem.</p>
+          <a role="button" href="https://forem.gitbook.io/forem-admin-guide" class="crayons-btn mt-1 w-100 self-end">Visit Guide</a>
+        </div>
+      </section>
+      <section class="crayons-card">
+        <div class="crayons-card__header">
+          <h3 class="crayons-subtitle-2">Forem Creators Community</h3>
+        </div>
+        <div class="crayons-card__body flex flex-wrap" style="min-height: 150px;">
+          <p>Meet other creators using Forem so that you can share tips with each other.</p>
+          <a role="button" href="https://creators.forem.com" class="crayons-btn mt-1 w-100 self-end">Join Community</a>
+        </div>
+      </section>
+      <section class="crayons-card">
+        <div class="crayons-card__header">
+          <h3 class="crayons-subtitle-2">Forem Community Experts</h3>
+        </div>
+        <div class="crayons-card__body flex flex-wrap" style="min-height: 150px;">
+          <p>Contact support to set up a free platform training session with a Forem expert.</p>
+          <a role="button" href="mailto:support@forem.com" class="crayons-btn mt-1 w-100 self-end">Contact Support</a>
```

Comment:
Based on Peter's suggestion we should just set these up manually with the community folks.

The UI is a bit different from the mock here:
![image](https://user-images.githubusercontent.com/11466782/108550099-b49f8e00-72b3-11eb-9e23-b61ab3135073.png)

Reply:
> we should just set these up manually with the community folks.

To confirm, this means that we don't need the user (the Forem Creator) to take any action then, right? The action would instead come from our side, the Forem bizdev side?

If that's right, then I think we just remove this bit from the design. When we came up with the design mocks in the original RFC (https://github.com/forem/rfcs/pull/39), it was under the assumption that all these links were ones we wanted to include; if that has changed, I think we can just remove this from the design given that we don't have anywhere to link to :)

Reply:
@jacobherrington let me know if you are blocked on this at all; i just want to make sure that you aren't waiting on any info for these links :)